src/video/SDL_video.c

```diff
@@ -4396,7 +4334,9 @@ void SDL_DestroyWindow(SDL_Window *window)
         _this->current_glwin = NULL;
     }
 
-    SDL_CompareAndSwapAtomicPointer(&_this->wakeup_window, window, NULL);
+    if (_this->wakeup_window == window) {
+        _this->wakeup_window = NULL;
+    }
```

Comment:
Are you rebasing to an older SDL3 revision ?  Otherwise what is this change for?